qemu/tests/cfg/nvdimm.cfg

```diff
@@ -83,3 +83,6 @@
             del dev_path
             del format_command
             del mount_command
+        - nvdimm_hotplug:
+            del mem_devs
```

Comment:
With this change, the `verify_nvdimm` is meaningless:
https://github.com/autotest/tp-qemu/blob/185b121e24d3f798e81d12e26cf1992822c1613f/qemu/tests/nvdimm.py#L46

Reply:
You are right. Updated this function, please help review again.